packages/core/src/http.interface.ts

```diff
@@ -3,9 +3,18 @@ import * as http from 'http';
 export interface HttpRequest extends http.IncomingMessage {
   body?: any;
   matchers?: string[];
+  route?: HttpRoute;
   [key: string]: any;
 }
 
+export type HttpParameters = Record<string, string | number>;
```

Comment:
I would also call it `RouteParameters` because it is not connected with `http` 😄